part2-06-python-functions.Rmd

```diff
@@ -4,7 +4,7 @@
 >
 >~Donald Knuth
 
-Functions (sometimes called “subroutines”) are arguably the most important concept in programming. We’ve already seen their use in many contexts, for example, when using functions like `len()` and `float()`. Here’s a bit of code that computes the GC content of a DNA sequence in a variable called `seq`:
+Functions (sometimes called "subroutines") are arguably the most important concept in programming. We’ve already seen their use in many contexts, for example, when using functions like `len()` and `float()`. Here’s a bit of code that computes the GC content of a DNA sequence in a variable called `seq`:
```

Comment:
We can maybe leave out the subroutines bit, that text was from back when I was trying to convert perl acolytes ;)

Reply:
Ha! Sounds good, updated in 5762f2aad39534e02a45a7df96d15fa88c1d2dba